mungegithub/mungers/release-note-label.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	"k8s.io/contrib/mungegithub/features"
+	"k8s.io/contrib/mungegithub/github"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	releaseNoteLabeler = "release-note-label"
+
+	releaseNote               = "release-note"
+	releaseNoteNeeded         = "release-note-needed"
+	releaseNoteNone           = "release-note-none"
+	releaseNoteBreakingChange = "release-note-breaking-change"
```

Comment:
This looks pretty much done except that we changed the `release-note-breaking-change` label to `release-note-action-required` a few days ago [here](https://github.com/kubernetes/kubernetes/pull/23315/files#diff-50af6ac44c5737b0c6bab6e296507507R184).  Can we get this submitted soon?  LGTM after that change.